playbooks/roles/mattermail/templates/config.json

```diff
@@ -0,0 +1,22 @@
+{
+    "Directory": "./data/",
+    "Profiles":[
+        {
+            "Name":     "{{ MATTERMOST_LOGIN }}",
```

Comment:
Nit: I'm not sure if this a standard thing or not to do in JSON, but are the extra spaces really warranted?

Reply:
I see that you got this from the README file on their repo. I guess they added spaces for easy initial demonstration.

Reply:
That's how Go formats indented code to improve readability. Not very orthodox, so I'll remove extra spaces.